### DISCUSSION TITLE
阿里云盘无法访问错误：临时解决方案（已测试OK）

### DISCUSSION THREAD

**Discussion by qizq2** [AlistGo/alist#3388] [Show and tell]
### Please make sure of the following things

- [X] I have read the [documentation](https://alist.nn.ci).
- [X] I'm sure there are no duplicate issues or discussions.
- [X] I'm sure it's due to `alist` and not something else(such as `Dependencies` or `Operational`).
- [X] I'm sure I'm using the latest version

### Alist Version / Alist 版本

v3.10.1

### Driver used / 使用的存储驱动

阿里云盘

### Describe the bug / 问题描述

阿里云盘接口更新导致所有用Alist挂载的阿里云盘基本都崩了，报错invalid X-Device-Id
**# 目前测试通过分享方式挂载的阿里网盘内容依然可用。**
建议各位获取新Token令牌或直接复制原挂载阿里云盘的Token，
**# 挂载阿里云盘分享的文件夹即可解决目前问题。**

### Reproduction / 复现链接

此问题已出现，原Issue #3375 内容过多，此贴仅用于置顶方法提醒各位Alist使用者可以用这个方法临时使用

**Comment by welcome** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by yb1229998088** [comment]
分享的没啥用呀，不支持压缩包

**Reply by duskNNNN** [reply]
把后缀改成支持的后缀格式，下载完改回来不就行了

**Reply by qizq2** [reply]
起码视频可以在线播放了，好多人用阿里云盘不就是为了在线看个视频嘛0.0

**Comment by rtyrhzhl** [comment]
希望开发者能及时更新![image](https://user-images.githubusercontent.com/124253713/218627210-c89fe6a2-c2f0-4ec8-8014-e6e43143d9c9.png)

**Reply by FayeLeeeeee** [reply]
请问下这个内测报名入口在哪里？

**Reply by rtyrhzhl** [reply]
> 请问下这个内测报名入口在哪里？

https://survey.alibaba.com/apps/zhiliao/I9Dd1Nl89

**Comment by xiaowang7212** [comment]
测试确实没问题，但是zip格式不可用 ipa格式也是看不到，因为官方不支持这些格式的分享·

**Comment by OtistarSun** [comment]
failed get link: UserDeviceIllegality:invalid X-Device-Id

**Comment by dreamsamethyst** [comment]
没有办法播放视频呀，没用啊，文件我也试了，下载不了

**Reply by qizq2** [reply]
分享文件夹是可以的，测试链接http://alist.xiaoya.pro/

**Comment by goofmi** [comment]
阿里云share 尝试了，也是不成功的，无法正常播放视频和浏览图片。

**Reply by qizq2** [reply]
目前我这边测试是OK的，测试链接http://alist.xiaoya.pro/

**Reply by yb1229998088** [reply]
最新版本不是已经修复了么，没必要再用分享的了吧，太麻烦
